### DISCUSSION TITLE
./therm_soln_compare.py

### DISCUSSION THREAD

**Discussion by wowodejiajia** [idaholab/moose#19577] [Q&A Tools] [answered]
Dear MOOSE developers,

I'm studying tutorials of heat_conduction. In the last step - Comparison of results.
It is executed by typing: ./therm_soln_compare.py
from the directory where the problems are run.
It can not generate an image called therm_soln_compare.png with line plots for these three cases.

**Comment by GiudGiud** [comment, marked as answer]
hello

just tried and it worked fine. 
Did you run all input files in the folder? Can you give us the output of `ls -l` in that folder ?
Here s the plot
![therm_soln_compare](https://user-images.githubusercontent.com/25037151/145139374-bc7301a8-dda9-40cb-8743-d6d3f9d76e9c.png)

**Reply by wowodejiajia** [reply]
-bash: ./therm_soln_compare.py: No such file or directory

**Reply by GiudGiud** [reply]
I saw the directory list in my email but it s not here??
the file should be there. Can you please give us the output of `ls -l` and `pwd` ?

this is the ./therm_soln_compare.py script btw
```
#!/usr/bin/env python
import matplotlib
matplotlib.use('Agg')
import matplotlib.pyplot as plt
import pandas as pd

data02a=pd.read_csv('therm_step02a_out_t_sampler_0006.csv')
data03=pd.read_csv('therm_step03_out_t_sampler_0006.csv')
data03a=pd.read_csv('therm_step03a_out_t_sampler_0006.csv')

plt.figure(figsize=(6.0,4.5))
ax=plt.gca()

data02a.plot(ax=ax, x='x', y='T',label='Conduction (step02a)')
data03.plot(ax=ax, x='x', y='T',label='Conduction + Time Deriv. (step03)')
data03a.plot(ax=ax, x='x', y='T',label='Conduction + Time Deriv. + Volumetric (step03a)')

ax.set_xlabel("Position")
ax.set_ylabel("Temperature")

plt.savefig('therm_soln_compare.png', bbox_inches='tight', pad_inches=0.1)
```

**Reply by wowodejiajia** [reply]
I don't have the therm_soln_compare.py file. Just put the files in the folder.
Thank you.
